src/tutorials/getting-started-with-drizzle-and-react.md

```diff
@@ -0,0 +1,792 @@
+
+## Testing the smart contract
+
+Before we proceed, we should write a couple tests to ensure that our contract works as expected.
+
+1. Create a new file named `MyStringStore.js` in the `test/` directory.
+
+1. Add the following content to the `MyStringStore.js` file:
+
+   ```javascript
+   const MyStringStore = artifacts.require("./MyStringStore.sol");
+
+   contract("MyStringStore", accounts => {
+     it("should store the string 'Hey there!'", async () => {
+       const myStringStore = await MyStringStore.deployed();
+
+       // Set myString to "Hey there!"
+       await myStringStore.set("Hey there!", { from: accounts[0] });
+
+       // Get myString from public variable getter
+       const storedString = await myStringStore.myString.call();
+
+       assert.equal(storedString, "Hey there!", "The string was not stored");
+     });
+   });
+   ```
+
+### Running the tests
+
+1. Back in the terminal, run the tests:
+
+   ```shell
+   truffle test
+   ```
+
+1. If all the tests pass, you'll see console output similar to this:
+
+   ```shell
+   Using network 'development'.
+
+     Contract: MyStringStore
+       ✓ should store the value 'Hey there!' (3085ms)
+
+     1 passing (3s)
+   ```
+
+Awesome! Now we know that the contract actually works.
+
+## Creating our React.js project
+
+Now that we are done with the smart contract, we can write our front-end client with React.js! In order to do this, simply run this command (if you have NPM version 5.2 or above):
+
+```shell
+npx create-react-app client
+```
+
+**If you have an older version of NPM**, make sure Create-React-App is installed globally as per the instructions in the [Setting up the development environment](#create-react-app) section and then run the following:
+
+```shell
+create-react-app client
+```
+
+This should create a `client` directory in your Truffle project and bootstrap a barebones React.js project for you to start building your front-end with.
+
+## Setting up the front-end client
+
+Now that we have a front-end client located inside the `client` directory, change into that directory with the command `cd client` and continue with the following steps to set it up.
+
+### Link up our build artifacts
+
+Since Create-React-App's default behavior disallows importing files from outside of the `src` folder, we need to bring the contracts in our `build` folder inside `src`. We can copy and paste them every time we compile our contracts, but a better way is to create a symlink.
+
+If you haven't created a symlink before, think of it as a magical portal in the file system that allows you act as if the file or folder is actually there.
+
+Let's change into the `src` directory and then create our symlinked folder:
+
+```shell
+// For MacOS and Linux
+
+cd src
+ln -s ../../build/contracts contracts
+
+// For Windows 7, 8 and 10
+// Using a Command Prompt as Admin
+
+cd src
+mklink \D C:\..\build\contracts contracts
```

Comment:
@DiscRiskandBisque Shouldn't this part be a relative link instead? Something like this:

```
mklink \D ..\..\build\contracts contracts
```

I don't have a Windows computer to test this on though.